### PR TITLE
updated code

### PR DESCRIPTION
**Pull request by Azizhamdani7**
please add

### CODE REVIEW

**Review by MuhammadAhmad665 — COMMENTED** [review]
please explane

File: backend/server.js [hunk 1]
```diff
@@ -10,9 +10,11 @@ const app = express();
 //middleware
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
-app.use(cors({
-  origin: ["http://localhost:3000/", ]
-}));
+app.use(
```
[the remark this answers is not in the document]
Comment (Azizhamdani7) [reply backend/server.js:13]: rearranged the code using prettier.

[the remark this answers is not in the document]
Comment (MuhammadAhmad665) [reply backend/server.js:13]: oh OK sure then i'll accept the PR now

[the remark this answers is not in the document]
Comment (Azizhamdani7) [reply backend/server.js:13]: great, thank you.

**Pull request merged**